dask/dataframe/rolling.py

```diff
@@ -39,10 +39,15 @@ def overlap_chunk(
     if isinstance(before, datetime.timedelta):
         before = len(prev_part)
 
+    expansion = out.shape[0] // combined.shape[0]
```

Comment:
Is this likely to cause issues with empty partitions? e.g. the one from

```python
In [33]: df = pd.DataFrame({"A": range(12), "B": [True] * 3 + [False] * 3 + [True] * 6})

In [34]: ddf = dd.from_pandas(df, 4)

ddf[df.B].get_partition(1).compute()
```

Reply:
I didn't find a test case to trigger this but I'm thinking a check here doesn't add much complexity and could prevent confusion later.